css/index.css

```diff
@@ -106,3 +107,148 @@ ul li a svg path {
 ul li a svg path:hover {
   fill: #fff;
 }
+
+.image-placeholder {
+  margin-left: 1%;
+  padding: 6%;
+  width: 88%;
+}
+
+.stroke {
+  border: 1px solid;
+  border-color: #1c1a19;
+  opacity: 0.5;
+  margin: 0% 16%;
+}
+
+.my-recent-work {
+  background-color: var(--white);
+}
+
+.title {
+  padding-top: 30%;
+  padding-bottom: 6%;
+  text-align: center;
+  font-family: var(--secondary-font-family);
+  font-size: 40px;
+  font-weight: 200;
+}
+
+.title-post {
+  font-family: var(--secondary-font-family);
+  margin-left: 6%;
+  font-weight: 200;
+  font-size: 43px;
+}
+
+.supporting-text {
+  margin: 3% 8% 5%;
+  font-family: Inter, sans-serif;
+  font-size: 22px;
+  font-weight: 300;
+}
+
+.tags {
+  display: flex;
+  justify-content: left;
+}
+
+.tag {
+  text-align: center;
+  border: 1.8px solid var(--black);
+  font-family: Inter, sans-serif;
+  margin-right: -23px;
+  padding: 2% 3%;
+  font-size: 19px;
+  margin-left: 8%;
+}
+
+.knowledge-btn {
+  border: none;
+  color: var(--white);
+  font-size: 18px;
+  padding: 4%;
+  background-color: rgb(255 255 255 / 28%);
+  font-family: Inter, sans-serif;
+}
+
+.knowledge-list {
+  padding: 3% 7%;
+}
+
+.btn {
+  background-color: var(--orange);
+  width: 36%;
+  height: 63px;
+  border: none;
+  color: var(--white);
+  font-family: Inter, sans-serif;
+  font-size: 22px;
+  margin: 3% 8%;
+}
+
+.professional-background {
+  color: var(--white);
+  background: linear-gradient(179deg, rgb(6 6 6 / 40.4%) 0.85%, rgb(19 19 19 / 92.9%) 84%),
+    url('../image/profession-placeholder.png');
+  background-repeat: no-repeat;
+  background-size: cover;
+  margin: 4% 5%;
+  background-position: top;
+  position: relative;
+}
+
+.professional-background:hover .professional-background-details {
```

Comment:
Right here, you should not be using a compound selector, instead give what you want to style an additional class and then use it in your css